cmd/kube-controller-manager/app/apps.go

```diff
@@ -33,6 +35,10 @@ import (
 )
 
 func startDaemonSetController(ctx context.Context, controllerContext ControllerContext) (controller.Interface, bool, error) {
+	controllerContext.ObjectOrMetadataInformerFactory.WithResource(appsv1.SchemeGroupVersion.WithResource("daemonsets"))
+	controllerContext.ObjectOrMetadataInformerFactory.WithResource(appsv1.SchemeGroupVersion.WithResource("controllerrevisions"))
+	controllerContext.ObjectOrMetadataInformerFactory.WithResource(corev1.SchemeGroupVersion.WithResource("pods"))
+	controllerContext.ObjectOrMetadataInformerFactory.WithResource(corev1.SchemeGroupVersion.WithResource("nodes"))
```

Comment:
hmm... this approach is really fragile to drift/typos/omission... I was expecting to either augment / wrap `controllerContext.InformerFactory` to track what typed informers were requested, or adjust the generated informerfactories to track group/version/resource of requested informers, and propagate that info to the started-typed-or-metadata factory